src/components/Sections/SectionHTML.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable react/no-danger */
+import './SectionHTML.less';
+import React from 'react';
+import PropTypes from 'prop-types';
+import DOMPurify from 'dompurify';
+
+const SectionHTML = ({ text, style }) =>
+  <div className="section-html" style={style} dangerouslySetInnerHTML={{ __html: DOMPurify.sanitize(text) }} />;
```

Comment:
The sanitize might be problematic, since the server already does its own, and this will override the user decision, I would remove this. Or make it configurable.